server/Dockerfile

```diff
@@ -6,13 +6,14 @@ VOLUME /timely-server-src
 VOLUME /timely
 
 RUN yum -y update && \
-  yum -y install java-1.8.0-openjdk-devel maven openssl apr && \
-  mkdir -p /timely/bin
+  yum -y install java-1.8.0-openjdk-devel maven openssl apr wget && \
+  mkdir -p /timely/bin && \
+  wget -O /usr/lib/jvm/jre/lib/ext/bcprov-jdk15on-155.jar https://www.bouncycastle.org/download/bcprov-jdk15on-155.jar && \
+  echo "security.provider.9=org.bouncycastle.jce.provider.BouncyCastleProvider" >> /usr/lib/jvm/jre/lib/security/java.security
```

Comment:
/etc/java/security/security.d/2000-org.bouncycastle.jce.provider.BouncyCastleProvider is created as part of the bouncy castle install, but timely still throws unless I add the provider to java.security.  I only did it this way originally because it is how the bouncy castle [wiki](http://www.bouncycastle.org/wiki/display/JA1/Provider+Installation) says to add statically.  I'm noob at these bits so just following instructions I find.